src/page-modules/assistant/details/trip-section/index.tsx

```diff
@@ -81,6 +85,19 @@ export default function TripSection({
           </TripRow>
         )}
 
+        {leg.transportSubmode === 'railReplacementBus' && (
```

Comment:
Is this something that could be modelled in the new flexible notification system @toretefre @rosvik ?

Reply:
Global messages are best for messages that are org specific and/or likely to change at some point, which I don't think this is. Have also tried to keep them "one collection of messages per page", not like this which would have one <GlobalMessages> per trip leg.

So I think hard-coding this is better.

Reply:
Good input! Thanks